tests/spread/store/validation-sets/task.yaml

```
@@ -0,0 +1,43 @@
+summary: test the validation-set commands
+
+environment:
+  SNAPCRAFT_ASSERTION_KEY: "$(HOST: echo ${SNAPCRAFT_ASSERTION_KEY})"
+  SNAPCRAFT_STORE_CREDENTIALS: "$(HOST: echo ${SNAPCRAFT_STORE_CREDENTIALS_STAGING})"
+
+prepare: |
+  if [[ -z "$SNAPCRAFT_STORE_CREDENTIALS" ]]; then
+    echo "No credentials set in env SNAPCRAFT_STORE_CREDENTIALS"
+    exit 1
```

```suggestion
    ERROR "No credentials set in env SNAPCRAFT_STORE_CREDENTIALS"
```

This doesn't need to include the trace.